L1Trigger/L1TTrackMatch/interface/L1TkEtMissEmuTrackTransform.h

```diff
@@ -0,0 +1,84 @@
+#ifndef L1Trigger_L1TTrackMatch_L1TkEtMissEmuTrackTransform_HH
+#define L1Trigger_L1TTrackMatch_L1TkEtMissEmuTrackTransform_HH
+
+#include "L1Trigger/L1TTrackMatch/interface/L1TkEtMissEmuAlgo.h"
+#include "DataFormats/L1TrackTrigger/interface/TTTypes.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+using namespace L1TkEtMissEmuAlgo;
+
+//Internal Word used by EtMiss Emulation, producer expects this wordtype
+//Allows for additional conversion functions e.g. from additional GTT emulators to this word
+struct InternalEtWord{
+    iZ0 pV;
+
+    iZ0 z0;
+    iPt pt;
+    iEta eta;
+    iglobPhi globalPhi;
+    iNstub nstubs;
+    ichi2bend bendChi2;
+    ichi2XY chi2rphidof;
+    ichi2XY chi2rzdof;
+
+    unsigned int Sector;
+
+    float phi;  //Used to debug cos phi LUT
+};
+
+// Converts float tracks to internal Et word including initial digitisation
+// Can also run directly from digitised TTTrack_Word  
+class L1TkEtMissEmuTrackTransform {
+public:
+  L1TkEtMissEmuTrackTransform() = default;  
+  ~L1TkEtMissEmuTrackTransform() {};
+
+  void GenerateLUTs();  //Generate internal LUTs needed for track transfrom
+
+  InternalEtWord TransformTrack(TTTrack< Ref_Phase2TrackerDigi_ >& track_ref,float PV);
+  InternalEtWord TransfromCuts(const edm::ParameterSet& iConfig);
+
+    //Converts local int phi to global int phi
```

Comment:
The spacing in front of some of the comments does not match the spacing in front of the code.

Reply:
Fixed in latest commit